### DISCUSSION TITLE
SuperTux 0.6.3

### DISCUSSION THREAD

**Discussion by tobbi** [SuperTux/supertux#2007] [Announcements]
The SuperTux team is excited to announce the release of SuperTux 0.6.3 after approximately 1.5 years of development.  This release introduces many new features; perhaps the most new features in a long time!

Some of the most significant changes include:
- WASM compilation! SuperTux nightlies can now be played directly in the browser (thanks to Semphris)
- Added swimming (thanks to Daniel and Zwatotem)
- Added walljumping (thanks to Daniel)
- Autotiles (thanks to Semphris)
- Updated animations (thanks to Alzter, Daniel and RustyBox)
- Refreshed many contrib worldmaps (thanks to Servalot)
- Many updates to paths, with easing, bezier curves, path sharing between objects, and more (thanks to Semphris)
- Remade the crystal tileset (thanks to Alzter and RustyBox)
- Many new snow tiles (thanks to Daniel)
- Many new objects, like the sideways bumper and falling blocks (thanks to Daniel)
- Custom particles (thanks to Semphris)
- A new rublight object (thanks to HybridDog)
- Official binaries for FreeBSD, Linux 32-bit and Ubuntu Touch (thanks to Semphris)
- Added in-game progress statistics (thanks to Semphris)
- New color picker based on OKLab (thanks to HybridDog)
- Add-on creator, to easily create add-on packages with your world (thanks to Semphris)
- Rework of the Revenge in Redmond worldmap (thanks to RustyBox)
- Added timeshift ambience in the worldmap (thanks to Semphris)
- Skippable cutscenes (thanks to Semphris)
- Editor auto-saves at regular intervals (thanks to Semphris)
- Optional integration with Discord (thanks to Semphris)
- Updated translations, of course (thanks to translators)

And these are just the most significant changes; there are plenty of other small features and bugfixes for you all to explore :)

Thank you, Merry Christmas, and happy SuperTuxing!!

<hr /><em>This discussion was created from the release <a href='https://github.com/SuperTux/supertux/releases/tag/v0.6.3'>SuperTux 0.6.3</a>.</em>

**Comment by Rami-Slicer** [comment]
Nice!

**Comment by invakid404** [comment]
@tobbi why is there no source tarball for this release? The GitHub one doesn't have the submodules included...

**Reply by Semphriss** [reply]
It is uploaded now.

**Reply by invakid404** [reply]
Oh, cool! Thanks!

**Comment by Frostwithasideofsalt** [comment]
🥳🥳🥳

**Comment by NIKIHVFG12** [comment]
HELLO ABLAN ARGENTINO JUGAMOS

**Comment by bergentroll** [comment]
So swimming was really surprising for regular player. Ability to swim in cold water seems natural for a penguin. Meanwhile it drastically changes mechanics on good old levels. Seems it is possible to deny swimming on "well known" levels e. g. by reason of bad water (too cold, with sharp shards of ice, poisoned, whatever). 

Glad to see the project is active. But also hope to meet a happy end of the story once, after all brand new features.
